packages/collaboration-manager/src/Operation.spec.ts

```diff
@@ -40,231 +40,6 @@ const createOperation = (
 
 
 describe('Operation', () => {
-  describe('.transform()', () => {
```

Comment:
Don't delete it, those tests still should pass even tho the functionality in another file now